Pickers/SWActionSheet.m

```diff
@@ -87,13 +87,25 @@ - (UIWindow *)window
     }
     else
     {
-        return SWActionSheetWindow = ({
-            UIWindow *window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
-            window.windowLevel        = self.windowLevel;
-            window.backgroundColor    = [UIColor clearColor];
-            window.rootViewController = [SWActionSheetVC new];
-            window;
-        });
+        UIWindow *window = nil;
+        if (@available(iOS 13.0, *)) {
+            UIScene *scene = [UIApplication sharedApplication].connectedScenes.allObjects.firstObject;
```

Comment:
I didn't check new things in ios13, jsut a quick check: 
accroding to https://noahgilmore.com/blog/uiwindowscene-black-screen/
we have to use `scene(scene:willConnectTo:options:)`
 sometimes. 
Probably ` UIScene *scene = [UIApplication sharedApplication].connectedScenes.allObjects.firstObject;` will solve all the rest options. But want to ask you double check it.

Reply:
It's a different behaviour. `scene(scene:willConnectTo:options:)` is called on the app initialisation, like the `application: didFinishLaunchingWithOptions ` method.

In the currently library, we don't let the user pass the active scene on which display the loader. Maybe this could be an improvement for the next releases?